build/flutter_app.gni

```diff
@@ -185,6 +187,20 @@ template("flutter_app") {
       flutter_tools_label,
     ]
   }
+
+  copy(target_name) {
+    sources = [
+      bundle_path,
+    ]
+
+    outputs = [
+      "$root_out_dir/$bundle_name",
+    ]
+
+    deps = [
+      ":$package_target_name",
+    ]
+  }
 }
 
 template("flutter_aot_app") {
```

Comment:
Should I add this to flutter_aot_app as well? What do you guys think of putting that template in its own file?